provider/aws/formation/rack.json

```diff
@@ -895,6 +896,11 @@
       "Description": "Dedicated Hardware",
       "Default": "default",
       "AllowedValues": [ "default", "dedicated" ]
+    },
+    "WhiteList": {
+      "Type": "String",
+      "Description": "Comma delimited list of CIDRs, e.g. `10.0.0.0/24,172.10.0.1/32`, to allow access to the rack api. Maximum 4 CIDR can be specified.",
```

Comment:
Why is only 4 CIDR supported?

Reply:
As you can see, I am adding the CIRD block in the ALB rule, they have restriction on that(Target Groups per Action per Application Load Balancer: 5): https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-limits.html
So one is host header, other 4 is for the source ip

Reply:
Why not updating Security Group?